src/background/index.ts

```diff
@@ -26,4 +26,19 @@ chrome.action.onClicked.addListener((tab) => {
   }
 });
 
+// Add context menu item
+chrome.contextMenus.create({
+  id: 'workdayCalendar',
+  title: 'Workday Calendar',
```

Comment:
Maybe it should be a verb like "View Workday Calendar". Up to you though.

<img width="455" alt="Screen Shot 2024-06-07 at 10 17 02 PM" src="https://github.com/mlool/workday-calendar-extension/assets/59376051/51ae32f3-ac66-4301-8d39-5d31ac3b99e1">

Reply:
Good call. Done!